examples/complete/main.tf

```diff
@@ -0,0 +1,36 @@
+provider "azurerm" {
+  version = "~>2.3"
+  features {}
+}
+
+locals {
+  unique_name_stub = substr(module.naming.unique-seed, 0, 5)
+}
+
+module "naming" {
+  source = "git::https://github.com/Azure/terraform-azurerm-naming"
+}
+
+resource "azurerm_databricks_workspace" "test_ws" {
+  name                      = local.unique_name_stub
+  resource_group_name       = var.resource_group_name
+  location                  = var.resource_group_location
+  sku                       = "premium"
+}
+
+# Force cluster deployment to wait to avoid state error
+resource "time_sleep" "wait_5_mins" {
```

Comment:
@LukeDevs how do you feel about having this defined outside the module? Feels like an internal detail to me.

Reply:
I could make this internal to the module and add a variable to change the time required in seconds...I think. Given the timing I'd prefer to add this to the back log as a small bit of tech debt/short-term future work though. @LukeDevs?

Reply:
[Expanding comment](https://github.com/Azure/terraform-databricks-sec-resources/pull/1/commits/6e0dbc8ae88995c1faa51f87d1aab1254cf30e20)

Reply:
Moved time_sleep inside module and tested. 👍